src/components/sections/About.js

```diff
@@ -140,44 +128,33 @@ const About = () => {
                 <Img fluid={data.art_pot.childImageSharp.fluid} />
               </Art>
               <div>
-                <h2>{t('Our community of experts is here for you')}</h2>
-                <p>
-                  {t(
-                    'If your teaching needs require custom digital experiences, our community of developers and educators are here to help.',
-                  )}
-                </p>
+                <h2>{t(ASSOCIATION.ABOUT_COMMUNITY_HEADER)}</h2>
+                <p>{t(ASSOCIATION.ABOUT_COMMUNITY_TEXT)}</p>
               </div>
             </Grid>
             <Grid>
               <div>
-                <h2>
-                  {t(
-                    'We enable open translational research in digital education',
-                  )}
-                </h2>
+                <h2>{t(ASSOCIATION.ABOUT_TRANSLATIONAL_HEADER)}</h2>
                 <p>
-                  <Trans>
-                    We provide a desktop application for anonymizing and
-                    analyzing learning analytics and data (
-                    <a
-                      href="https://insights.graasp.org"
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      Graasp Insights
-                    </a>
-                    ), and a repository to share collections of open datasets (
-                    <a
-                      href="https://library.graasp.org"
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      Graasp Library
-                    </a>
-                    ). This enables the research community and public
-                    institutions to conduct evidence-based assessments of
-                    innovative pedagogical scenarios and technological
-                    solutions.
+                  <Trans
+                    components={[
+                      <a
+                        href="https://insights.graasp.org"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        Graasp Insights
+                      </a>,
+                      <a
+                        href="https://library.graasp.org"
```

Comment:
Should we factor out these links?

Reply:
Sure! Same as above? Where do you suggest I factor this out to?